web/src/components/toolbar/use/useMetabolite.ts

```diff
@@ -0,0 +1,155 @@
+import {
+  PropType, computed, watch, Ref, ref,
+} from '@vue/composition-api';
+import { colors } from '../../../utils/constants';
+
+// TODO this is a subset of a type from Vuex
```

Comment:
I think it might be time for a `types.ts` file that contains common types like this that are referenced frequently. Thoughts?

Reply:
Fine with me.